src/tools/jsondocck/src/main.rs

```diff
@@ -199,9 +207,15 @@ fn check_command(command: Command, cache: &mut Cache) -> Result<(), CkError> {
                     let val = cache.get_value(&command.args[0])?;
                     match select(&val, &command.args[1]) {
                         Ok(results) => {
-                            let pat: Value = serde_json::from_str(&command.args[2]).unwrap();
-
-                            !results.is_empty() && results.into_iter().any(|val| *val == pat)
+                            // FIXME: Share the pat getting code with the `Is` branch.
+                            let v_holder;
+                            let pat: &Value = if command.args[2].starts_with("$") {
+                                &cache.variables[&command.args[2][1..]]
+                            } else {
+                                v_holder = serde_json::from_str(&command.args[2]).unwrap();
+                                &v_holder
+                            };
```

Comment:
Do you mind doing that here? It looks like the code is exactly the same.

Reply:
Done in ba22a69d964283d388a362830e73a992f024a6bf